tools/test-run-dev

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+from __future__ import absolute_import
+
+import os
+import sys
+import time
+import signal
+import subprocess
+
+from six.moves import range
+
+
+if __name__ == '__main__':
+    print("Testing development server start!")
+
+    logfile = open('/tmp/run-dev-output', 'w+')
+
+    TOOLS_DIR = os.path.dirname(os.path.abspath(__file__))
+    args = ["{}/run-dev.py".format(TOOLS_DIR)]
+    run_dev = subprocess.Popen(args, stdout=logfile, stderr=logfile)
+
+    found = False
+    key = "Quit the server with CTRL-C."
+    for i in range(10):
+        time.sleep(5)
```

Comment:
Probably should sleep in shorter increments (maybe 0.5s?) so there's less latency.